libs/blocks/quiz/quiz.css

```diff
@@ -417,11 +409,19 @@ html[dir="rtl"] .quiz-step::after {
   .quiz-option.has-icon .quiz-option-image {
     display: flex;
   }
+
```

Comment:
is there a reason why those rules are being moved? i think changes like that should be avoided to make clearer PR and file histories ...

Reply:
Thanks for taking the time to check this out. One of the tasks in this PR involved revamping breakpoints. It looks like some simple refactoring for that effort. I've opened another PR to resolve the concerns I can.
https://github.com/adobecom/milo/pull/1636

Reply:
Thanks, @npeltier, for the find, and @fullcolorcoder for the fix. It's has been merged into uar-integration